src/content/en/2024/cookies.md

```diff
@@ -1,16 +1,820 @@
+</figure>
+
+### Privacy & security risks
+**Web tracking.** Cookies are used by third parties to track users across websites and record their browsing behavior and interests. In targeted advertising, this data is leveraged to show users advertisements aligned with their interest. This tracking usually takes place the following way; third-party code embedded on a site can set a cookie that identifies a user. Then, the same third-party can record user activity by obtaining that cookie back when the user visits other websites where it is embedded as well (see also the [Privacy](./privacy) chapter). We note that first-party cookies can also be used for online tracking, methods such as cookie syncing allow to bypass the limitation of third-party cookies and track users [across different websites](https://dl.acm.org/doi/abs/10.1145/3442381.3449837).
+
+**Cookie theft and session hijacking.** Cookies are used to store session information such as credentials (session token) for authentication purposes across several HTTP requests. However, if these cookies were to be obtained by a malicious actor, they could use them to authenticate to the corresponding web servers. If cookies are not properly set by web servers, they could be prone to cross-site vulnerabilities such as [session hijacking](https://developer.mozilla.org/en-US/docs/Glossary/Session_Hijacking), cross-site request forgery ([CSRF](https://developer.mozilla.org/en-US/docs/Web/Security/Practical_implementation_guides/CSRF_prevention)), cross-site script inclusion ([XSS](https://developer.mozilla.org/en-US/docs/Glossary/Cross-site_scripting)), and others (see also the [Security](./security) chapter).
+
+### Caveats
+
+You can learn more about the methodology applied by the HTTP Archive for the Web Almanac in 2024 on the [Methodology](./methodology) page. There are limitations to that methodology which may impact the results in this chapter:
+- Data is collected by automatically visiting websites in a non-interactive way; user interaction could modify the way websites set and use cookies in practice. For example, HTTP Archive's tools do not interact with cookie banners (if any) and so cookies that would be set after interaction with these banners are not observed by our study.
+- Websites are visited from servers located in the US that have no cookie set when each independent website visit starts; this is quite different from a user accumulating and saving web cookies while browsing the web. The location from which visits are performed can impact cookie behavior due to regulation and legislation such as [GDPR](https://gdpr-info.eu/).
+- For each website, the home page is visited as well as one other page from the same website.
+- Most of the results presented in this chapter are based on the top one million (top 1M by CrUX rank) most visited websites that were successfully reached during the HTTP Archive crawl of June 2024.
```

Comment:
This is no longer true is it? Think we reran for ALL the sites. So I'll update this in the final edit I'm doing now.

Reply:
Yes, leave as is: 
- I only regenerated the first figures 1st/3rd party prevalence across all ranks
- I would not have had enough time to re-extract all stats, re-plot everything, and update the text content while also formatting and reviewing the chapter for today. 
- Given that the top 1M websites capture about 95% of all page loads of web users according to [this measurement paper](https://dl.acm.org/doi/pdf/10.1145/3517745.3561418) on Google's private telemetry dataset, I focused on finalizing what we had and left a comment for future editions to maybe run the other queries not only on top 1M websites. 

Thanks!